backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/util/DB.java

```diff
@@ -539,6 +528,21 @@ public CPreparedStatement prepareStatement(final String sql,
 											   final int resultSetType,
 											   final int resultSetConcurrency,
 											   final String trxName)
+	{
+		{
+			if (dbPerformanceMonitoringHelper.isPerfMonActive())
```

Comment:
do we still need this since we are measuring at QueryStatisticsLogger?

Reply:
https://github.com/metasfresh/metasfresh/pull/13721#discussion_r1016312726